lib/runtime/wazero/imports.go

```diff
@@ -2303,27 +2304,19 @@ func ext_storage_root_version_1(ctx context.Context, m api.Module) uint64 {
 	return rootSpan
 }
 
-func ext_storage_root_version_2(ctx context.Context, m api.Module, version uint32) uint64 { //skipcq: RVV-B0012
+func ext_storage_root_version_2(ctx context.Context, m api.Module, _ uint32) uint64 { //skipcq: RVV-B0012
```

Comment:
Is it weird that we have an unused argument for an imported func?

Reply:
Yeah, this argument can be ignored for this host func since it uses the same `state_version` value from the wasm `runtime_version` structure, and in Gossamer we cache the `runtime_version`. But the plan is to use the parameter as soon as trie refactor takes place.